### ISSUE TITLE
unittest failure: test_runSimulationTranslated

### ISSUE THREAD

**Issue by thorade** [lbl-srg/BuildingsPy#124]
On Ubuntu, when I run `python buildingspy/tests/test_examples_dymola.py` it fails with error message:
```
.Model name       = Buildings.Controls.Continuous.Examples.PIDHysteresis
Output directory = case1
Time             = Thu Feb 23 11:16:04 2017

Model name       = Buildings.Controls.Continuous.Examples.PIDHysteresis
Output directory = case2
Time             = Thu Feb 23 11:16:04 2017

.Model name       = Buildings.Controls.Continuous.Examples.PIDHysteresis
Output directory = case2
Time             = Thu Feb 23 11:16:13 2017

Model name       = Buildings.Controls.Continuous.Examples.PIDHysteresis
Output directory = case1
Time             = Thu Feb 23 11:16:13 2017

*** Error: Killing simulation in /tmp/tmp-simulator-mthorade-8roOVl/BuildingsPy.
*** Error: Killing simulation in /tmp/tmp-simulator-mthorade-8roOVl/BuildingsPy.
*** Error: Killing simulation in /tmp/tmp-simulator-mthorade-8roOVl/BuildingsPy.
*** Error: Simulation failed in '/tmp/tmp-simulator-mthorade-8roOVl/BuildingsPy'
   You need to delete the directory manually.
E
======================================================================
ERROR: test_runSimulationTranslated (__main__.Test_example_dymola_runSimulation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "buildingspy/tests/test_examples_dymola.py", line 42, in test_runSimulationTranslated
    s.main()
  File "/home/mthorade/git/BuildingsPy/buildingspy/examples/dymola/runSimulationTranslated.py", line 59, in main
    po.map(simulateTranslatedModel, li)
  File "/usr/lib/python2.7/multiprocessing/pool.py", line 251, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/usr/lib/python2.7/multiprocessing/pool.py", line 567, in get
    raise self._value
IOError: File /tmp/tmp-simulator-mthorade-8roOVl/BuildingsPy/simulator.log does not exist

----------------------------------------------------------------------
Ran 3 tests in 71.314s

FAILED (errors=1)
```
This is for me not a new problem, see e.g. https://github.com/lbl-srg/BuildingsPy/pull/98#issuecomment-264451762
This is the only test that fails, would be nice to get it running on my machine also.

The simulation gets killed after some timeout, and I am not really sure why.
Maybe it is related to license problems? I am using a network license.
Or is it related to the `setUp`method in `test_examples_dymola.py`? It mentions a path that does not exist on my machine.

**Comment by thorade** [comment]
When setting `s1.showGUI(True)` I can see that the simulation fails, because the library and thus also the model is not loaded. Probably the path to the library is wrong!?

**Comment by mwetter** [comment]
@thorade The path to the library is wrong, will look into it.

**Comment by thorade** [comment]
This seems to be fixed by #127 
I can now successfully run 
`make unittest` 
as well as 
`python buildingspy/tests/test_examples_dymola.py`
on Xubuntu 16.04 with Python2.
